src/GitHub.App/Models/PullRequestModel.cs

```diff
@@ -3,7 +3,6 @@
 using GitHub.Primitives;
 using GitHub.VisualStudio.Helpers;
 using System.Diagnostics;
-using System.Collections.Generic;
```

Comment:
Nit: Would be nice to completely remove the changes to these unchanged files to give a cleaner diff ;)